### PR TITLE
Add Name and NameTest classes

### LINKED ISSUE #217: No validation is done on name strings. Implement a Name class to fix this
The Name class should contain a String as a name, but check for symbols or numbers in the String object supplied to its constructor. If any are found, it should throw an InvalidDataException.

### PR DESCRIPTION
**Pull request by matheril**
Name contains a name, but checks for invalid characters in the String provided to its constructor.
Names can only contain alphabetical characters.

Fixes #217

### CODE REVIEW

**Review by froststein — APPROVED** [review]
Looks good to merge

File: src/main/java/seedu/splitlah/ui/Message.java [hunk 1]
```diff
@@ -157,4 +157,6 @@
     // Activity List Command
     public static final String ERROR_ACTIVITYLIST_ACTIVITY_EMPTY =
             "There are currently no activities in this session";
+    public static final String ERROR_NAME_INVALIDNAME =
+            "Invalid name. Names must contain only alphabetical characters.";
```
Comment (warrencxw) [inline src/main/java/seedu/splitlah/ui/Message.java:161]: Consider putting messages for Name class separately as with other classes, with its own section and inline comment.

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

File: src/main/java/seedu/splitlah/ui/Message.java (hunk 1 above)
Comment (matheril) [reply src/main/java/seedu/splitlah/ui/Message.java]: Resolved

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Pull request merged**